src/lib.rs

```diff
@@ -56,6 +44,20 @@ impl ImageWrapper {
 
         array
     }
+
+    fn channels<'a>(&self, py: Python<'a>, names: Vec<String>) -> PyResult<Vec<&'a PyArray2<f32>>> {
```

Comment:
Let us name this `channels_with_names` or something like that to keep the `channels` function available for later - it could return all the channels as some kind of object/dict that contains the name. But no need for that now.

Reply:
Done